packages/react/src/components/Button/Button.tsx

```diff
@@ -2,10 +2,20 @@ import '@compassion-gds/css/src/components/Button/button.css';
 
 export interface ButtonProps {
   label: string;
+  mode?: 'primary' | 'secondary' | 'tertiary',
+  size?: 'small' | 'medium' | 'large',
+  disabled?: boolean,
+  onClick?: () => void,
 }
 
-const Button = (props: ButtonProps) => {
-  return <button className="button-main">{props.label}</button>;
+const Button = ({ label = '', mode = 'primary', size = 'medium', disabled = false, onClick }: ButtonProps) => {
```

Comment:
If `label` has a default value e.g. `label = ''` it's considered an optional parameter

A prop is optional if we have a fallback for it

Reply:
Maybe we can just change to `label?: string` for the interface

Reply:
Not sure, I was thinking to remove the default value, so the label is always required.

Reply:
Ok, removed the default value, it should always have a `label` to look as expected.